frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/review-page.tsx

```diff
@@ -13,66 +14,91 @@ const ReviewPage: React.FC<ReviewPageProps> = ({ state }) => {
     tier2BackingStore,
     tier1Policy,
     tier2Policy,
+    namespacePolicyType,
+    bucketClassType,
+    readNamespaceStore,
   } = state;
   const { error, isLoading } = state;
   const { t } = useTranslation();
 
+  const getReviewForNamespaceStore = () => (
+    <>
+      <ReviewListBody>
+        <span>{t('ceph-storage-plugin~Namespace Policy: ')}</span>&nbsp;
+        <span className="text-secondary">{namespacePolicyType}</span>
+      </ReviewListBody>
+      {namespacePolicyType === NamespacePolicyType.SINGLE && (
+        <ReviewListBody>
+          <span>{t('ceph-storage-plugin~Read and write NamespaceStore : ')}</span>&nbsp;
+          <span className="text-secondary">{readNamespaceStore[0]?.metadata.name}</span>
```

Comment:
Why only show one read name space store name?

Reply:
read and write namespace store are same for a single bucket class. I have declared it as an array as we can use this in next PR for multi namespace bucket class.